src/shared/lib/StoreUtils.ts

```diff
@@ -1622,6 +1623,113 @@ export function makeGetOncoKbCnaAnnotationForOncoprint(
     }
 }
 
+export function getSampleClinicalDataMapByThreshold(
+    clinicalData: ClinicalData[],
+    clinicalAttributeId: string,
+    threshold: number
+) {
+    return _.reduce(
+        clinicalData,
+        (acc, next) => {
+            if (next.clinicalAttributeId === clinicalAttributeId) {
+                const value = getNumberClinicalDataValue(next);
+                if (value && value >= threshold) {
+                    acc[next.sampleId] = next;
+                }
+            }
+            return acc;
+        },
+        {} as { [key: string]: ClinicalData }
+    );
+}
+
+export function getNumberClinicalDataValue(
+    clinicalData: ClinicalData
+): number | undefined {
+    if (Number.isNaN(clinicalData.value)) {
+        return undefined;
+    } else {
+        return Number(clinicalData.value);
+    }
+}
+
+export function getSampleNumberClinicalDataValue(
```

Comment:
Same as above. `numerical` instead of `number`?
```suggestion
export function getSampleNumericalClinicalDataValue(
```